dvc/scm/base.py

```diff
@@ -24,6 +24,16 @@ class RevError(SCMError):
     pass
 
 
+class NOSCMError(SCMError):
+    def __init__(self):
+        msg = (
+            "Only supported for Git repositories. If you're\n"
+            "seeing this error in a Git repo, try updating the DVC "
+            "configuration with\n`dvc config core.no_scm False`."
```

Comment:
Doesn't seem like we need newlines here. Also slight adjustment for `False`
```suggestion
            "Only supported for Git repositories. If you're "
            "seeing this error in a Git repo, try updating the DVC "
            "configuration with `dvc config core.no_scm false`."
```